diagrams.net/src/worker.js

```diff
@@ -27,6 +29,32 @@ export class Worker {
     this.convertTimeout = process.env.KROKI_DIAGRAMSNET_CONVERT_TIMEOUT || '15000'
   }
 
+  async resolveImage(svgRoot) {
+    let $ = cheerio.load(svgRoot,  {
```

Comment:
Since we are using Puppeteer we could potentially parse/edit the SVG in `page.evaluate`? That would remove the dependency on `cheerio`.

Reply:
Definitely I am not a JS expert I will check to change it

Reply:
Did you find a way to implement this feature without relying on cheerio? I can take a look if you want?